pages.de/common/g++.md

```diff
@@ -1,4 +1,4 @@
-# gplusplus
+# g++
```

Comment:
While checking the other discussion I remembered that `tldr-lint` doesn't support symbols in page titles yet so unless we add it, this would be a blocker in this PR.